job-runner/chain/traverser.go

```diff
@@ -201,7 +201,20 @@ JOB_REAPER:
 
 		// If the job did not complete successfully, then ignore subsequent jobs.
 		// For example, if job B of A -> B -> C  fails, then C is not ran.
-		if doneJ.State != proto.STATE_COMPLETE {
+		if doneJob.State != proto.STATE_COMPLETE {
+			// TODO(alyssa): Add supported for retrying nested sequences. This only
+			// retries current sequence we are in.
+			if t.chain.CanRetrySequence(doneJob) {
+				jLogger.Info("job did not complete successfully. retrying sequence.")
+				t.sendJL(doneJob, nil)
```

Comment:
I don't think this is needed. The job-runner will always create a job log when it runs a job. The only reason we need to create job logs in the traverser is when we fail to create a job runner or some higher-level thing like that

Reply:
ah yes the runner does create a job log. ok i'll remove this and update the runner to include the sequence related fields. good catch - thanks.